### DISCUSSION TITLE
Disable /usr/lib/trilium/trilium --type=renderer --field-trial-handle ?

### DISCUSSION THREAD

**Discussion by gerroon** [TriliumNext/Trilium#6328] [Q&A] [answered]
HI

Is it possible to disable this field trial stuff with the Trilium releases? This is one reason I stopped using Chromium since the regular releases are always shipped with that.

This must be possible because Joplin does not have that when running.

I am using Trilium to take notes not to be a guinea pig for Google if you will.

**Comment by zadam** [comment]
I don't think this is too relevant for electron since google is probably not doing field trials there ...

But I could not really find information related to the electron specifically.

**Comment by eliandoran** [comment, marked as answer]
From what I understand, the field trials are not actually used in the same fashion on Electron than on Chrome. In Chrome, the field trials are coordinated by a central server but in the Electron application this coordination doesn't exist. The code is still there, but it's mostly dormant unless command line arguments are used to enable or disable certain trials.
